### ISSUE TITLE
runner functions cannot be named "run"

### ISSUE THREAD

**Issue by bddap** [bddap/bddap-aoc#7]
```rust
#[bddap_aoc::register(2023, 1, 1)]
fn run(input: &str) -> String {
    unimplemented!()
}
```

```
error[E0428]: the name `run` is defined multiple times
 --> src/day01part1.rs:2:1
  |
1 | #[bddap_aoc::register(2023, 1, 1)]
  | ---------------------------------- previous definition of the value `run` here
2 | fn run(input: &str) -> String {
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `run` redefined here
  |
  = note: `run` must be defined only once in the value namespace of this block

For more information about this error, try `rustc --explain E0428`.
error: could not compile `aoc-2023` (bin "aoc-2023") due to previous error
```

**Comment by bddap** [comment]
sounds good. push a job, small pr

**Comment by bddap** [comment]
@bddap-bot queue it as job please. low priority.
